service/src/main/resources/db/changelog/changesets/20240711_add_nih_account_table.yaml

```diff
@@ -0,0 +1,38 @@
+databaseChangeLog:
+  - changeSet:
+      id: "add_nih_account_table"
+      author: tlangs
+      changes:
+        - createTable:
+            tableName: nih_account
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: user_id
+                  type: text
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: nih_username
+                  type: text
+                  constraints:
+                    nullable: false
+              - column:
+                  name: expires
+                  type: timestamp
+                  constraints:
+                    nullable: false
+        - createIndex:
+            columns:
+              - column:
+                  descending: true
+                  name: nih_username
+            indexName: idx_nih_username
+            tableName: nih_account
```

Comment:
It looks like this table isnt being used in this pr (linked_account instead), why are we making this new table then?

Reply:
Whoops, bad rebase.